Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:18-alpine3.20
+
+
+WORKDIR /app/
+
+EXPOSE 3000 3306
+
+VOLUME [ "/app/mounted-host-files" ]
+
+COPY package*.json .
+
+COPY .env.example .env
+
+# - attempt to automate creation of .env file and needed data by copying and appending to .env.example on build time
```

Comment:
The attempt is failing because it's not the right way of doing things with docker compose.

The right way is to have an `.env` file which is loaded by docker compose, and which then makes the environment variables available to the process.

`.env` should be a part of `.dockerignore` in fact!